Emby.Server.Implementations/HttpServer/HttpListenerHost.cs

```diff
@@ -230,7 +231,8 @@ private int GetStatusCode(Exception ex)
             switch (ex)
             {
                 case ArgumentException _: return 400;
-                case SecurityException _: return 401;
+                case AuthenticationException _: return 401;
+                case SecurityException _: return 403;
                 case DirectoryNotFoundException _:
                 case FileNotFoundException _:
```

Comment:
I feel like these two could be another error or something, they don't usually have the same meaning.